src/app/app.component.html

```diff
@@ -1,4 +1,3 @@
 <app-header></app-header>
-<app-breadcrumbs></app-breadcrumbs>
```

Comment:
You should use custom structural directive to show/hide the breadcrumbs. Breadcrumbs component should be inside the app.component.